modules/ephemeral-storage-sharing-configmaps-across-namespaces.adoc

```diff
@@ -0,0 +1,36 @@
+:_content-type: PROCEDURE
+
+[id="ephemeral-storage-sharing-configmaps-across-namespaces_{context}"]
+= Sharing a config map across namespaces
+
+[role="_abstract"]
+To share a config map across namespaces in a cluster, you create a `SharedConfigMap` object for that config map.
+
+.Prerequisites
+
+You have privileges to perform the following actions:
+
+* Create instances of the `sharedsecrets.sharedresource.openshift.io` CRD at a cluster-scoped level.
```

Comment:
IIUC we are talking about getting sharedconfigmap here, no? @rolfedh so can we replace `sharedsecrets.sharedresource.openshift.io` CRD with `sharedconfigmaps.sharedresource.openshift.io` 

Please correct/guide me if I am wrong

Reply:
Nice catch. My error. Updated. Thanks.